### DISCUSSION TITLE
How do i add more words to python query for the same output

### DISCUSSION THREAD

**Discussion by justindiaz7474** [community/community#21500] [Other Feature Feedback, Questions, & Ideas]
<p>I understand that you can duplicate the output with each of which with different words but is there anyway i can combine words with the same outcome in my virtual assitant? it’ll save so much space.</p>
<pre><code>elif 'where is' in query:
</code></pre>
<p>^^^^^^ How do i add more words to activate the command</p>

**Comment by justindiaz7474** [comment]
<p>The problem im having is that if i add another one to the query, Then the virtual assitant just repeats the response in a loop…EX. </p>
<pre><code>if 'hi' or 'hello' in query:
</code></pre>

**Comment by fire-eggs** [comment, marked as answer]
<p>See my reply to your original post.</p>
